test/data/scroller.js

```diff
@@ -0,0 +1,20 @@
+var items = [
+  {
+    "text": "Did Pluto have rivers and lakes?<br clear='all'/><br/><br/>",
```

Comment:
@stulees Those tags are just there for testing purposes to show that HTML is stripped out and not drawn to the canvas.